docs/w3lib.rst

```diff
@@ -26,3 +26,9 @@ w3lib Package
 
 .. automodule:: w3lib.url
     :members:
+
+:mod:`form` Module
+-----------------
+
+.. automodule:: w3lib.form
```

Comment:
I'm not sure we should document it, because the only function in this module is deprecated

Reply:
Oh sorry that makes sense to not add it, i'll update